geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/utils/ThreadManagement.scala

```diff
@@ -11,15 +11,23 @@ package org.locationtech.geomesa.index.utils
 import java.io.Closeable
 import java.util.concurrent.{ScheduledFuture, ScheduledThreadPoolExecutor, TimeUnit}
 
-import com.typesafe.scalalogging.LazyLogging
+import com.typesafe.scalalogging.Logger
+import org.locationtech.geomesa.filter.filterToString
+import org.locationtech.geomesa.utils.collection.CloseableIterator
 import org.locationtech.geomesa.utils.concurrent.ExitingExecutor
+import org.locationtech.geomesa.utils.iterators.ExceptionalIterator
+import org.opengis.filter.Filter
+import org.slf4j.LoggerFactory
 
+import scala.concurrent.duration.Duration
 import scala.util.control.NonFatal
 
 /**
  * Singleton for registering and managing running queries.
  */
-object ThreadManagement extends LazyLogging {
+object ThreadManagement {
+
+  private val logger = Logger(LoggerFactory.getLogger(ThreadManagement.getClass.getName.replace("$", "")))
```

Comment:
Does this let us use 'org.locationtech.geomesa.index.utils.ThreadManagement' in log4j properties files?

Reply:
right. spark actually has a little utility that does that automatically for objects :heart:

Reply:
Really?!  Neat.  

I mean, could one gin up a ObjectLazyLogging trait?:)